book/src/getting_started.md

```diff
@@ -11,9 +11,7 @@ It explains how to compile and run the `hello-word` example to verify your setup
    On Ubuntu, the following is sufficient:
 
 ```sh
-apt install build-essential curl git python3 pkg-config \
-  libssl-dev llvm-dev cmake libclang-dev gcc-arm-none-eabi \
-  clang unzip lld ninja-build
+apt install git rustup ninja-build pkg-config libudev-dev clang gcc-arm-none-eabi
```

Comment:
according to [their instructions](https://probe.rs/docs/getting-started/installation/#debian-based-linux) probe-rs also requires cmake and pkg-config.

Reply:
The alternative is to us the binstall instructions from probe-rs, that drops libudev-dev from this list, but adds `cargo-binstall` to the deps. No strong opinion either way from me.

Reply:
The alternative is to us the binstall instructions from probe-rs, that drops libudev-dev from this list, but adds `cargo-binstall` to the deps. No strong opinion either way from me.

Reply:
The alternative is to us the binstall instructions from probe-rs, that drops libudev-dev from this list, but adds `cargo-binstall` to the deps. No strong opinion either way from me.